contrib/kubernetes/flowlogs-pipeline.conf.yaml

```diff
@@ -5,6 +5,7 @@ parameters:
     collector:
       hostname: 0.0.0.0
       port: 2055
+      portLegacy: 2056
```

Comment:
@joel

This file is automatically generated by running `confgenerator`  from the Makefile `https://github.com/netobserv/flowlogs-pipeline/blob/main/Makefile#L243` 

The parameter needs to be added to https://github.com/netobserv/flowlogs-pipeline/blob/main/pkg/confgen/flowlogs2metrics_config.go#L56 

And then running the makefile should generate exactly this output 

BTW: It is using the same API structures:: look here:  https://github.com/netobserv/flowlogs-pipeline/blob/main/pkg/confgen/config.go#L41


Sorry, a bit more work :-(